src/app/views/query-response/snippets/snippets-helper.tsx

```diff
@@ -30,10 +31,12 @@ interface ISupportedLanguages {
 
 export function renderSnippets(supportedLanguages: ISupportedLanguages) {
   const sortedSupportedLanguages: ISupportedLanguages = {};
-  Object.keys(supportedLanguages).sort().forEach(key => {
+  sortedSupportedLanguages.CSharp = supportedLanguages.CSharp;
+  Object.keys(supportedLanguages).filter((language) => language !== 'CSharp').sort().forEach(key => {
```

Comment:
It might be easier to use the [`sort(compareFn)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) version of the sort which will save having to filter out csharp first.

Reply:
I don't see how the `compareFn` makes it easier.. I see the alternative as just splicing then unshifting...

Reply:
I was thinking:

```javascript
Object.keys(supportedLanguages).sort((l1, l2) => {
  if (l1 == 'CSharp') {
    return -1;
  }
  if (l2 == 'CSharp') {
    return 1;
  }
  if (l1 < l2) {
    return -1;
  } else if (l1 > l2) {
    return 1;
  }
  return 0;
})
```